docs/cimpl6ToolingReference.md

```diff
@@ -0,0 +1,692 @@
+* [modeldoc](#model-documentation-export) - this directory contains files that present the model with a look and feel similar to [Javadoc](#https://www.oreilly.com/library/view/learn-to-program/9781680500523/f_0126.html)
+
+## FHIR Export
+
+The FHIR export consists of profiles, extensions, value sets, code systems, examples, and other artifacts needed to create a FHIR IG. These are not the _pages_ of the IG, but rather various definitions that underlie the IG presentation. You must run the [IG publisher](#creating-the-implementation-guide) to create the IG itself.
+
+The FHIR export appears in the /fhir subdirectory under the designated "out" directory. The directory contains further subdirectories for code systems, extensions, the IG (guide), the logical model (logical), profiles, and value sets.
+
+### Logical Model Export
+
+The logical model is an alternative representation of the CIMPL model, expressed in a familiar FHIR-like format. The logical model closely follows the CIMPL definitions because it reflects the model _prior to_ mapping to FHIR resources. Each entry is presented as if it were a new FHIR resource, rather than a profile. The logical model is formally expressed as a set of FHIR `StructureDefinitions`. These definitions are distinguished from profiles because the names generated by the tooling end in `-model`.
+
+Because the logical model does not entail the additional complication of mapping, it is useful for review purposes. The logical model can be checked for clinical relevance and sufficiency before making mapping decisions.
+
+An optional part of the FHIR export, the logical model can be turned on or off using the `includeLogicalModels` flag in the configuration file.
+
+#### _Logical Model Example_
+![Logical Model Example](img_cimpl/logical-model-example.png)
+
+## Model Documentation Export
+
+The model documentation provides another way to view the logical model. This view emphasizes the class hierarchy defined in CIMPL, the datatypes, associated constraints, and relationships between the classes (references). The panel in the upper left allows the user to filter on the namespace and (in SHR-CLI 6.6 and higher) the class type (Entry, Group, or Element). The right-hand panel shows details for the selected class, include the full set of properties and constraints. 
+
+For many users, especially those with experience in object-oriented modeling, the model documentation is the quickest way to navigate around the model.
+
+#### _Model Documentation Example_
+
+![Model Documentation Example](img_cimpl/model-doc-example.png)
+
+## Data Dictionary Export
+
+The Data Dictionary is an Microsoft Excel file containing a simplified, flattened list of model elements and value sets, extracted from the FHIR profiles. The spreadsheet has five tabs:
+
+* The first tab contains a key to the columns in the other tabs;
+* The second tab contains a list of the primary profiles and their descriptions;
```

Comment:
Modify to say, `The second tab, **Profiles**`...

Reply:
done